Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -10,6 +10,7 @@ import Storage
 private let log = Logger.browserLogger
 
 private let ThumbnailIdentifier = "Thumbnail"
+private let FrecencyLimit = 20
```

Comment:
Yeah, that's why I mentioned "maximized for both portrait and landscape" — i.e., use screen dimensions instead of frame, and figure both orientations at once.

The reason why I think this is worth doing: we don't just use this number for `numberOfItemsInSection`, we also use it as the limit argument to `getSitesByFrecencyWithLimit`, so this directly affects the number of returned results. It probably doesn't save a ton of time on the inner queries, but it does avoid us e.g., fetching favicons for pages that we'll never show.

Or, put differently: set this to 1000 and see how slow it goes!